ludwig/backend/ray.py

```diff
@@ -244,7 +245,7 @@ def tune_batch_size_fn(
     # inside of a RayTrainer class, so we manually set the local_rank to 0 so that it picks up the right
     # device to tune batch size on.
     initialize_pytorch(local_rank=0, local_size=_local_size())
-    distributed = get_current_dist_strategy(allow_local=True)()
+    distributed = init_dist_strategy("local")
```

Comment:
Will this still be `local` after having merged https://github.com/ludwig-ai/ludwig/pull/2934?

Reply:
Yes, good catch, we resolved this when merging the distirubted batch size tuning pr, so should be good:

https://github.com/ludwig-ai/ludwig/blob/master/ludwig/backend/ray.py#L249